src/dnddiceroller/Die.java

```diff
@@ -8,5 +10,21 @@
  * @author Paul Scarrone
  */
 public class Die {
-  
+    private int die;
+    private int value;
+    private Random dieSide = new Random();
+   
+    public Die(int die) {
+       this.die = die;
+       
 }
+
+    public void roll() {
+        value = dieSide.nextInt(die) + 1;// dieSide is equal to 0 thri (die -1).
```

Comment:
since you remarked out the `System.out.println....` you should delete it and the `value` here since you don't need it :+1: